tools/doxygen_warning_patterns.txt

```diff
@@ -0,0 +1,9 @@
+adc.hpp
```

Comment:
I'll merge this and delete this file for you.

Reply:
Thanks